docs/github/v2/15-deployment/android.md

```diff
@@ -39,17 +79,25 @@ end
 ```ruby
 # fastlane/Fastfile
 platform :android do
-  desc "Upload a new Android version to the Google Play Store"
-  lane :internal do
-    upload_to_play_store(track: 'internal',release_status: 'completed',aab: "#{ENV['ANDROID_BUILD_FILE_PATH']}")
+  desc "Upload a new Android version to the production Google Play Store"
+  lane :production do
+    upload_to_play_store(track: 'production', release_status: 'completed', aab: "#{ENV['ANDROID_BUILD_FILE_PATH']}")
   end
-  lane :alpha do
-    upload_to_play_store(track: 'alpha',release_status: 'completed',aab: "#{ENV['ANDROID_BUILD_FILE_PATH']}")
+
+  desc "Upload a new Android internal version to Google Play"
+  lane :internal do
+    upload_to_play_store(track: 'internal', release_status: 'completed', aab: "#{ENV['ANDROID_BUILD_FILE_PATH']}")
   end
 end
 ```
 
-### 4- Add jobs to your GitHub workflow
+If you would like to upload to other tracks (namely, `alpha` or `beta`), you can create additional lanes that look the same as the `:production` and `:internal` lanes except for the name and `track` parameter. Additionally, if you want to create draft releases (which will be necessary until you've manually pushed at least one build through the manual submission process), you will want to switch `release_status` from `completed` to `draft`.
+
+### 6- Add jobs to your GitHub workflow
+
+The following workflow establishes two jobs. The first builds your game into an AAB file, and the second uploads that generated bundle to the Play Store.
+
+If you instead intend to manually distribute your game outside the Play Store as an APK, remove the `androidAppBundle` flag from the `with` section of the `game-ci/unity-builder@v2` action in the workflow. You can also remove the `androidKeystoreName`, `androidKeystoreBase64`, `androidKeyStorePass`, `androidKeyaliasName`, and `androidKeyaliasPass` properties. From there, you'll want to remove the entire `releaseToGooglePlay` job. If you just want to download the APK, it will be available in the "artifacts" section of a completed workflow run, or you can add a new job that performs whatever custom upload logic you'd like.
```

Comment:
I don't think we need this paragraph, as it has already been established that this guide is specifically for deployment to the Google Play Store.

Reply:
Totally agree, removed